File: C/Dynamic_Memory_Allocation/Reversed_array.c
#include<stdio.h>

int main()
{
	int i, n, *p;
	printf("Input number of elements of array: ");
	scanf("%d", &n);
	int a = n;	
	p = (int*)malloc(n*sizeof(int));
	
	for(i = 0; i < n; i++)
	{
		printf("Array[%d] = ", i);
		scanf("%d", (p + i));
	}
	
	i = 0;
	while(i < n)
	{
		int tempt = *(p + i);
		*(p + i) = *(p + (n - 1));
		*(p + (n - 1)) = tempt;
		i++;
		n--;
	}
	
	printf("-----------------------------\n");
	printf("The reversed array:\n");
	for(i = 0; i < a; i++)
	{
		printf("Array[%d] = %d\n", i, *(p + i));
	}
}

File: Python/Modules/NumPy/Array/Reversed array.py
import numpy as np

List = []
size = int(input("Input number of elements: "))

for i in range(size):
    element = int(input("Array[" + str(i) + "] = "))
    List.append(element)

Array = np.array(List)

print("The original array:", Array)
List.reverse()
Array = np.array(List)
print("The reversed array", Array)
